compiler/zrc_diagnostics/src/diagnostic_kind.rs

```diff
@@ -111,6 +111,10 @@ pub enum DiagnosticKind {
     NumberLiteralOutOfBounds(String, String, String, String),
     #[error("global variable initializer must be a constant expression")]
     GlobalInitializerMustBeConstant,
+
+    // PREPROCESSOR ERRORS
```

Comment:
Explicitly list the errors the preprocessor generates, not just a single generic String error. `zrc_diagnostics` might be used by external crates.